### PR TITLE
Fix versions of security bugs related to jackson-databind, gson, and protobuf-java components

### PR DESCRIPTION
**Pull request by lilai23**
【Fix issue】#1373

【Modification】Fix version of security vulnerabilities related to jackson-databind, gson, protobuf-java components

[Use case description] None

[Self-test situation] 1. Local static check and cleanup passed

[Scope of influence] None

### CODE REVIEW

File: sermant-integration-tests/dubbo-test/pom.xml [hunk 1]
```diff
@@ -67,6 +67,17 @@
...
+                    <exclusion>
+                        <groupId>com.google.code.gson</groupId>
+                        <artifactId>gson</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+            <dependency>
+                <groupId>com.google.code.gson</groupId>
+                <artifactId>gson</artifactId>
+                <version>2.8.9</version>
```
Comment (luanwenfei-venus) [inline sermant-integration-tests/dubbo-test/pom.xml]: 版本没有统一管理

File: sermant-plugins/sermant-service-registry/spring-cloud-registry-plugin/pom.xml [hunk 2]
```diff
@@ -71,6 +71,17 @@
...
+                <exclusion>
+                    <groupId>com.google.code.gson</groupId>
+                    <artifactId>gson</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>2.8.9</version>
```
Comment (luanwenfei-venus) [inline sermant-plugins/sermant-service-registry/spring-cloud-registry-plugin/pom.xml]: 版本没有同意管理，并且范围不是provide了

File: sermant-integration-tests/dubbo-test/pom.xml (hunk 1 above)
Comment (lilai23) [reply sermant-integration-tests/dubbo-test/pom.xml]: 已修复

File: sermant-plugins/sermant-service-registry/spring-cloud-registry-plugin/pom.xml (hunk 2 above)
Comment (lilai23) [reply sermant-plugins/sermant-service-registry/spring-cloud-registry-plugin/pom.xml]: 已修复

**Review by hanbingleixue — APPROVED** [review]

**Review by luanwenfei-venus — APPROVED** [review]

**Pull request merged**